diff --git a/ranger/gui/widgets/view_miller.py b/ranger/gui/widgets/view_miller.py
index 55d401a0..a27907b2 100644
--- a/ranger/gui/widgets/view_miller.py
+++ b/ranger/gui/widgets/view_miller.py
@@ -138,9 +138,12 @@ class ViewMiller(ViewBase):  # pylint: disable=too-many-ancestors,too-many-insta
         if 'outline' in border_types:
             try:
                 # pylint: disable=no-member
-                win.hline(0, left_start, curses.ACS_HLINE, right_end - left_start)
-                win.hline(self.hei - 1, left_start, curses.ACS_HLINE, right_end - left_start)
-                win.vline(1, left_start, curses.ACS_VLINE, self.hei - 2)
+                # win.hline(0, left_start, curses.ACS_HLINE, right_end - left_start)
+                win.hline(0, left_start, "-", right_end - left_start)
+                # win.hline(self.hei - 1, left_start, curses.ACS_HLINE, right_end - left_start)
+                win.hline(self.hei - 1, left_start, "-", right_end - left_start)
+                # win.vline(1, left_start, curses.ACS_VLINE, self.hei - 2)
+                win.vline(1, left_start, "|", self.hei - 2)
                 # pylint: enable=no-member
             except curses.error:
                 pass
@@ -158,13 +161,18 @@ class ViewMiller(ViewBase):  # pylint: disable=too-many-ancestors,too-many-insta
                 y = self.hei - 1
                 try:
                     # pylint: disable=no-member
-                    win.vline(1, x, curses.ACS_VLINE, y - 1)
+                    # win.vline(1, x, curses.ACS_VLINE, y - 1)
+                    win.vline(1, x, "|", y - 1)
                     if 'outline' in border_types:
-                        self.addch(0, x, curses.ACS_TTEE, 0)
-                        self.addch(y, x, curses.ACS_BTEE, 0)
+                        # self.addch(0, x, curses.ACS_TTEE, 0)
+                        self.addch(0, x, "+", 0)
+                        # self.addch(y, x, curses.ACS_BTEE, 0)
+                        self.addch(y, x, "+", 0)
                     else:
-                        self.addch(0, x, curses.ACS_VLINE, 0)
-                        self.addch(y, x, curses.ACS_VLINE, 0)
+                        # self.addch(0, x, curses.ACS_VLINE, 0)
+                        self.addch(0, x, "|", 0)
+                        # self.addch(y, x, curses.ACS_VLINE, 0)
+                        self.addch(y, x, "|", 0)
                     # pylint: enable=no-member
                 except curses.error:
                     # in case it's off the boundaries
@@ -174,17 +182,22 @@ class ViewMiller(ViewBase):  # pylint: disable=too-many-ancestors,too-many-insta
             # Draw the last vertical line
             try:
                 # pylint: disable=no-member
-                win.vline(1, right_end, curses.ACS_VLINE, self.hei - 2)
+                # win.vline(1, right_end, curses.ACS_VLINE, self.hei - 2)
+                win.vline(1, right_end, "|", self.hei - 2)
                 # pylint: enable=no-member
             except curses.error:
                 pass
 
         if 'outline' in border_types:
             # pylint: disable=no-member
-            self.addch(0, left_start, curses.ACS_ULCORNER)
-            self.addch(self.hei - 1, left_start, curses.ACS_LLCORNER)
-            self.addch(0, right_end, curses.ACS_URCORNER)
-            self.addch(self.hei - 1, right_end, curses.ACS_LRCORNER)
+            # self.addch(0, left_start, curses.ACS_ULCORNER)
+            self.addch(0, left_start, "+")
+            # self.addch(self.hei - 1, left_start, curses.ACS_LLCORNER)
+            self.addch(self.hei - 1, left_start, "+")
+            # self.addch(0, right_end, curses.ACS_URCORNER)
+            self.addch(0, right_end, "+")
+            # self.addch(self.hei - 1, right_end, curses.ACS_LRCORNER)
+            self.addch(self.hei - 1, right_end, "+")
             # pylint: enable=no-member
 
     def _collapse(self):
